lib/shoulda/matchers/active_record/association_matcher.rb

```diff
@@ -1443,18 +1449,12 @@ def primary_key_correct?(klass)
         end
 
         def foreign_key
```

Comment:
Should we rename this `actual_foreign_key`? I'm reading through this code (AssociationMatcher has gotten really large 😞 ) and I'm wondering if it's going to be confusing when comparing this to `options[:foreign_key]`.

Reply:
I agree. I've thought about splitting into smaller files, but I still have doubts. If you have a suggestion, I'm all ears.

Reply:
Done!